config/rbac/instascale_role.yaml

```diff
@@ -65,19 +49,10 @@ rules:
 - apiGroups:
   - machine.openshift.io
   resources:
-  - '*'
-  verbs:
-  - create
-  - delete
-  - get
-  - list
-  - patch
-  - update
-  - watch
-- apiGroups:
-  - workload.codeflare.dev
-  resources:
-  - appwrappers
+  - controlplanemachinesets
+  - machinehealthchecks
```

Comment:
Are these two resources really used by InstaScale?

Reply:
I should have specified I've pushed this as myself and mark are testing which are actually used and what we can cut away.

Reply:
My understanding is that InstaScale only uses the Machine and MachineSet APIs but better double checking.